packages/design-system/scripts/cmsds-migrate/src/lib/migrate-helpers.mjs

```diff
@@ -0,0 +1,173 @@
+import chalk from 'chalk';
+import { globby } from 'globby';
+import inquirer from 'inquirer';
+import path from 'path';
+import { readdir, readFile, writeFile } from 'node:fs/promises';
+
+// confirm prompt for starting operations, display warning and y/n dialog
+export const confirmStart = () => {
+  return new Promise((resolve, reject) => {
+    inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'yesno',
+        prefix: `${chalk.red('!!')} Please be sure you are running this in a git repo. ${chalk.red('!!\n!!')} Modifications will occur to files listed above.    ${chalk.red('!!\n\n')}${chalk.green('?')}`,
+        message: 'Start modification?',
+        default: true,
+      }])
+      .then(choice => { resolve(choice.yesno) })
+      .catch(err => { reject(err) })
+  })
+}
+
+// return array of all globby pattern matches in one level
+export const doPatternSearch = async (config) => {
+  let res = []
+  for (const p of config.patterns) {
+    res.push(...await getGlob(p, config.globbyConfig).catch(err => error('doPatternSearch: '+err)))
+  }
+  return res
+}
+
+// error and quit
+export const error = (message) => {
+  console.log(`\n${chalk.red('!!')} Error: ${message}\n`)
+  process.exit(1)
+}
+
+export const findMatchedFiles = async (fileList, config) => {
+  return getAllFileContents(fileList, config.globbyConfig.cwd).then((globbed) => {
+    // filter out minified files
+    const filtered = globbed.filter(f => {
+      return (f.data.match(/[\n\r]/g) || []).length > 3
+    })
+    .filter(f => {
+      config.expressions.forEach(e =>  {
+        if (e.from.test(f.data)) f.matches++
+      })
+      return f.matches > 0
+    })
+    
+    const matchedFiles = filtered.reduce((prev, current) => {
+      const fn = path.relative(config.globbyConfig.cwd, current.file)
+      return [...prev, fn]
+    }, [])
+
+    return matchedFiles
+  })
+}
+
+// read file contents of all matched files, return filename and data, default matches (0)
+export const getAllFileContents = (fileList, cwd) => {
+  const readPromises = fileList.map(async file => {
+    file = cwd + path.sep + file;
+    return readFile(file, 'utf8')
+      .then(data => {
+        return { file: file, data: data, matches: 0 }
+      }).catch(err => error('getAllFileContents: '+err))
+  })
+  return Promise.all(readPromises)
+}
+
+// return list of files from path
+export const getConfigFileList = (path) => {
+  return new Promise((resolve, reject) => {
+    readdir(path)
+      .then(files => {
+        resolve(files)
```

Comment:
Looks like the `inquirer` `choices` can also take a objects of `{ name, value }`. Without going into reading these files for their descriptions, what if we renamed the files to
```
migrate-html-buttons.mjs 
cleanup-migration-notices.mjs 
migrate-react-buttons.mjs
```
and then replaced hyphens with spaces?

```suggestion
        resolve(
          files.map(file => ({
            name: file.replaceAll('-', ' ').split('.')[0], 
            value: file 
          }))
        )
```

Tried this, and it works. File renaming is optional because they still make sense

Reply:
Sure that makes it a little easier to read.